packages/filebrowser/style/base.css

```diff
@@ -196,38 +197,38 @@
  * Use container queries (not yet supported by our version of stylelint)
  * to display either a small or large header for the last-modified column.
  */
-/* stylelint-disable */
```

Comment:
Looks like we also need to keep this comment:

![image](https://github.com/user-attachments/assets/3b646559-39a5-448f-812b-58a806cf0302)